pkg/ephemeral/ephemeral_test.go

```diff
@@ -77,19 +83,39 @@ func (s *EphemeralSuite) TestRegisterPodOptionsApplier(c *check.C) {
 	}
 }
 
+type TestPodApplier struct{}
+
+func (TestPodApplier) Apply(*corev1.Pod) error { return nil }
+
+func (s *EphemeralSuite) TestRegisterPodApplier(c *check.C) {
+	var applier TestPodApplier
+
+	c.Assert(len(ephemeral.Pod), check.Equals, 0)
```

Comment:
nit: There's a more expressive `check.HasLen` checker
```suggestion
	c.Assert(ephemeral.Pod, check.HasLen, 0)
```